### PR TITLE
Stop using pkg/scheduled_test

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Comment by kevmoo** [comment]
Ping, @munificent ?

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,3 @@
-.idea
 .pub
-packages
 .packages
-build
\ No newline at end of file
+pubspec.lock
```
Comment (munificent) [inline .gitignore]: I do check in the lockfile. It doesn't affect consumers of dart_style, but it helps ensure everyone hacking on it gets the same experience.

File: pubspec.lock [hunk 2]
```diff
@@ -1,335 +0,0 @@
-# Generated by pub
```
Comment (munificent) [inline pubspec.lock]: Let's not ditch this file.

File: test/io_test.dart [hunk 3]
```diff
@@ -9,146 +9,126 @@ import 'dart:io';
...
-    var badBefore;
-    var goodBefore;
+    DateTime badBefore;
+    DateTime goodBefore;
 
-    schedule(() {
-      badBefore = modTime('bad.dart');
-      goodBefore = modTime('good.dart');
+    badBefore = modTime('bad.dart');
+    goodBefore = modTime('good.dart');
```
Comment (munificent) [inline test/io_test.dart]: May as well unify these with the above two lines:

```dart
var badBefore = modTile('bad.dart');
var goodBefore = modTime('good.dart');
```

File: pubspec.lock (hunk 2 above)
Comment (kevmoo) [reply pubspec.lock]: ack

File: test/io_test.dart (hunk 3 above)
Comment (kevmoo) [reply test/io_test.dart]: yup

**Pull request merged**